Sources/AppStoreConnectCLI/Services/Operations/InviteTesterOperation.swift

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct InviteTesterOperation: APIOperation {
+
+    struct InviteBetaTesterDependencies {
+        let apps: (APIEndpoint<AppsResponse>) -> Future<AppsResponse, Error>
+        let betaGroupsResponse: (APIEndpoint<BetaGroupsResponse>) -> Future<BetaGroupsResponse, Error>
+        let betaTesterResponse: (APIEndpoint<BetaTesterResponse>) -> Future<BetaTesterResponse, Error>
+    }
+
+    private enum InviteTesterError: LocalizedError {
+        case noGroupsExist(groupNames: [String], bundleId: String)
+
+        var errorDescription: String? {
+            switch self {
+            case .noGroupsExist(let groupNames, let bundleId):
+                return "One or more of beta groups \"\(groupNames)\" don't exist or don't belong to application with bundle ID \"\(bundleId)\"."
+            }
+        }
+    }
+
+    private let options: InviteBetaTesterOptions
+
+    private let getAppsOperation: GetAppsOperation
+
+    init(options: InviteBetaTesterOptions) {
+        self.options = options
+
+        self.getAppsOperation = GetAppsOperation(options: .init(bundleIds: [options.bundleId]))
```

Comment:
You probably don't need to retain this operation.  You could create it in the `execute` method.